### DISCUSSION TITLE
Handle paths to `best_model.h5` when passing in models to `sleap-track`

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1480] [Ideas]
Right now we get an obscure error message if the full path to the model checkpoint (`best_model.h5`) is provided to `sleap-track` as opposed to the model folder.

We already handle the case where we pass in the path to the `training_config.json` or `initial_config.json` as opposed to the checkpoint. Let's just add path handling for the checkpoint as well to avoid confusion.

Related: #1478

**Comment by gitttt-1234** [comment]
Closing this discussion, moved this to `sleap-nn`: https://github.com/talmolab/sleap-nn/issues/575
